casper/network.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, validator_set, protocol=BlockchainProtocol, force_justify_mes
     # async network rework
     #
     def delay(self, sender, receiver):
+        # return 1
         return r.choice([i for i in range(3)])
```

Comment:
r.randint(0, 2) should be faster (INSANE efficiency gainz lol), and a bit more readable.

Reply:
lolz